src/frontend/test_runner/tests/testdata/subquery_expr.yaml

```diff
@@ -130,6 +130,18 @@
           LogicalProject { exprs: [(count >= 1:Int32)] }
             LogicalAgg { aggs: [count] }
               LogicalValues { rows: [[1:Int32]], schema: Schema { fields: [:Int32] } }
+- sql: |
+    select a + 1, b::varchar, c from (values (1, 2, 3), (4, 5, 6)) t(a, b, c);
+  logical_plan: |
+    LogicalProject { exprs: [($0:Int32 + 1:Int32), $1:Int32::Varchar, $2:Int32] }
+      LogicalValues { rows: [[1:Int32, 2:Int32, 3:Int32], [4:Int32, 5:Int32, 6:Int32]], schema: Schema { fields: [:Int32, :Int32, :Int32] } }
+- sql: |
+    select sum(a), max(b + c + 10), string_agg(c::varchar || '~', ',') from (values (1, 2, 3), (4, 5, 6)) as t(a, b, c);
+  logical_plan: |
+    LogicalProject { exprs: [sum($0:Int32), max((($1:Int32 + $2:Int32) + 10:Int32)), string_agg(ConcatOp($2:Int32::Varchar, '~':Varchar), ',':Varchar)] }
```

Comment:
Related to the ambiguous input ref issue, I think this is exceedingly confusing with the  positional input refs. In this case, it's even worse than with a named column, as $0 should refer to the output of prev column, but here it does not.

Reply:
Maybe rather than position, we should use the anonymous column name `?column?_0`, as it will not be confused with a positional ref.

Reply:
Note that anonymous column name still suffers from: https://github.com/singularity-data/risingwave/issues/4455

I think we should specify both name and positional ref if available (as e.g. `name:$0`). If name not available, the inputref gets displayed as e.g. `:$0`

Reply:
Btw, we already use anonymous column name in the output for the rows. 

```sql
select sum(a), max(b + c + 10), string_agg(c::varchar || '~', ',') from (values (1, 2, 3), (4, 5, 6)) as t(a, b, c);
 ?column? | ?column? | ?column? 
----------+----------+----------
        5 |       21 | 3~,6~
```

Reply:
So what about we using `?value?_0`, to make it clear that it's a ref to a _value_ column?

Reply:
@jon-chuang After some thinking and discussion with @BugenZhao, I believe we'd better align with PG, that is, `*VALUES*_0.column_0` for the first column of the first `VALUES`. Any further comments?

Reply:
Sounds good!